src/testHelpers.spec.ts

```diff
@@ -42,3 +44,17 @@ export function trim(strings: TemplateStringsArray, ...args) {
     }
     return trimLeading(text);
 }
+
+/**
+ * Test that the given object has zero diagnostics. If diagnostics are found, they are printed to the console in a pretty fashion.
+ */
+export function expectZeroDiagnostics(obj: { getDiagnostics(): BsDiagnostic[] }) {
```

Comment:
@elsassph here's a helper function to make the weird tests mentioned in [your comment here](https://github.com/rokucommunity/brighterscript/pull/252#discussion_r544958176) a little less confusing. This will print out a much nicer set of all diagnostics, their lines, the bs code, and range. While also making the usage nicer as well:

```
it('has no diagnostics', ()=>{
    expectZeroDiagnostics(program);
})```